torch/_meta_registrations.py

```diff
@@ -525,6 +536,40 @@ def meta_index_Tensor(self, indices):
     return self.new_empty(before_shape + replacement_shape + after_shape)
 
 
+@register_meta([aten.add.Tensor], register_dispatcher=False)
+def meta_add(self, other, *, alpha=1):
+    check(torch.is_tensor(self), lambda: f"expected self to be tensor but got {type(self)}")
+    out_shape = self.shape
+    if torch.is_tensor(other):
+        out_shape = _broadcast_shapes(self.shape, other.shape)
+    _, out_dtype = elementwise_dtypes(
+            self, other, type_promotion_kind=ELEMENTWISE_TYPE_PROMOTION_KIND.DEFAULT
+        )
+    return self.new_empty(out_shape, dtype=out_dtype)
+
+@register_meta([aten.add_.Tensor], register_dispatcher=False)
+def meta_add_(self, other, *, alpha=1):
+    self = meta_add(self, other, alpha=alpha)
+    return self
+
+
+@register_meta([aten.convolution_backward.default])
+def meta_convolution_backward(grad_output_, input_, weight_, bias_sizes_opt, stride, padding, dilation, transposed, output_padding, groups, output_mask):
+    # TODO: THIS IS SUPER WRONG AND WOULD WORK FOR SOME MODELS!!!!
```

Comment:
@ezyang yeah actually hitting an error due to the use of `.options()` which is invalid for FakeTensors

Reply:
If you can figure out how to fix it that would be great